src/components/Main.tsx

```diff
@@ -348,7 +348,7 @@ const EdgeApp = () => {
       screenOptions={{
         drawerPosition: 'right',
         drawerType: 'front',
-        drawerStyle: { backgroundColor: 'transparent', bottom: 0 },
+        drawerStyle: { backgroundColor: 'transparent', bottom: 0, flexDirection: 'row', justifyContent: 'flex-end' },
```

Comment:
The problem is that the component has a set width. This can be fixed with `width: undefined` and then we need to position the component to the right with `right: 0`. Add a `alignItems: 'stretch'` for the SideMenu to be stretched out (or add `flexGrow: 1` back to the SideMenu, perhaps). The only problem is there is 56 pixels mis-alignement for some reason when the Drawer is open for some reason, so `right: 56` is needed.

This approach will not only position the SideMenu visually but also the drawer itself is positioned correctly. Otherwise the drawer continues to have the same width and this effects tap areas.

Alternatively, if we don't want to change the width and mess with that, and we're okay with having the tap-areas remain as they are, then the simpler fix is:

```
drawerStyle: { backgroundColor: 'transparent', bottom: 0, alignItems: 'flex-end' }
```

No need to change the orientation because we can use `align` instead of `justify`.